### PR TITLE
fix(CSS): add references to counter style specification and a tool

### LINKED ISSUE #27539: List-style-type needs a pointer to Ready-made Counter Styles
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

Information about custom counter styles.

### What did you expect to see?

The article at https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style has a section entitled [Ready-made counter styles](https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style#ready-made_counter_styles) which should probably also be added to this article about list-style-type, as it is relevant and useful in both locations.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/list-style-type`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/list-style-type/index.md
* Last commit: https://github.com/mdn/content/commit/73091fbe590d96857d743eaeec5aee4a8101994f
* Document last modified: 2023-02-21T05:50:15.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- Fixes https://github.com/mdn/content/issues/27539

### CODE REVIEW

File: files/en-us/web/css/list-style-type/index.md [hunk 1]
```diff
@@ -171,6 +171,8 @@ Note that:
 - `disclosure-closed`
   - : Symbol indicating that a disclosure widget, like {{HTMLElement("details")}} is closed.
 
+Details of all the avilable counter styles used by various culters around the world can be found in [Ready-made Counter Styles](https://www.w3.org/TR/predefined-counter-styles/) document.
```
Comment (hamishwillee) [inline files/en-us/web/css/list-style-type/index.md:174]: Just passing through. Presumably there is no MDN doc for this you could have linked.

```suggestion
Details of all the available counter styles used by various cultures around the world can be found in [Ready-made Counter Styles](https://www.w3.org/TR/predefined-counter-styles/) document.
```

Comment (OnkarRuikar) [reply files/en-us/web/css/list-style-type/index.md]: I thought of adding the link to the spec-urls.

Comment (hamishwillee) [reply files/en-us/web/css/list-style-type/index.md]: Yes. I guess I'm saying spec urls are usually a last resort to link to. But if there isn't anything better on MDN then good call.

Either way. Suggestion above.

File: files/en-us/web/css/list-style-type/index.md [hunk 2]
```diff
@@ -171,6 +171,8 @@ Note that:
 - `disclosure-closed`
   - : Symbol indicating that a disclosure widget, like {{HTMLElement("details")}} is closed.
 
+Details of all the available counter styles used by various cultures around the world can be found in [Ready-made Counter Styles](https://www.w3.org/TR/predefined-counter-styles/) document.
```
Comment (estelle) [inline files/en-us/web/css/list-style-type/index.md:174]: we should include a link to the example which includes an example of all the listed values.

Comment (OnkarRuikar) [reply files/en-us/web/css/list-style-type/index.md:174]: Our example covers only 65 values whereas the spec document covers 177. For example, [kashmiri](https://www.w3.org/TR/predefined-counter-styles/#kashmiri) is not there in the example.

**Review by estelle — APPROVED** [review]

**Comment by chrisdavidmills** [comment]
Thanks for the contribution @OnkarRuikar, this is super useful.

**Pull request merged**